samples/drivers/watchdog/sample.yaml

```diff
@@ -48,7 +48,19 @@ tests:
   sample.drivers.watchdog.stm32h7_wwdg:
     extra_args: DTC_OVERLAY_FILE=boards/stm32h7_wwdg.overlay
     filter: dt_compat_enabled("st,stm32-window-watchdog")
-    platform_allow: nucleo_h743zi
+    platform_allow:
+      - nucleo_h723zg
+      - nucleo_h745zi_q
+      - stm32h7s78_dk
+      - nucleo_h743zi
+      - stm32h735g_disco
+      - nucleo_h753zi
+      - stm32h750b_dk
+      - stm32h7b3i_dk
+      - stm32h745i_disco
+      - nucleo_h755zi_q
+      - stm32h747i_disco
+      - nucleo_h7a3zi_q
```

Comment:
What about filtering on `SOC_SERIES_STM32H7X` ?

Reply:
`filter: dt_compat_enabled("st,stm32-window-watchdog") and CONFIG_SOC_SERIES_STM32H7X`
does not filter correctly
neither
`filter: CONFIG_SOC_SERIES_STM32H7X and CONFIG_WWDG_STM32`